src/wallet/wallet2.cpp

```diff
@@ -166,6 +167,23 @@ std::unique_ptr<tools::wallet2> make_basic(const boost::program_options::variabl
   auto daemon_host = command_line::get_arg(vm, opts.daemon_host);
   auto daemon_port = command_line::get_arg(vm, opts.daemon_port);
 
+  int major, minor;
+  auto subaddress_lookahead = command_line::get_arg(vm, opts.subaddress_lookahead);
+
+  if (!subaddress_lookahead.empty()) 
+  {
+    std::string Mm = subaddress_lookahead;
+    auto pos = Mm.find(":");
+    THROW_WALLET_EXCEPTION_IF(pos == std::string::npos, tools::error::wallet_internal_error, tools::wallet2::tr("invalid subaddress lookahead format"));
+    major = std::stoi(Mm.substr(0, pos)); 
+    minor = std::stoi(Mm.substr(pos+1,Mm.length()));
+    THROW_WALLET_EXCEPTION_IF(major>255, tools::error::wallet_internal_error, tools::wallet2::tr("major must be less than 255"));
+    THROW_WALLET_EXCEPTION_IF(minor>255, tools::error::wallet_internal_error, tools::wallet2::tr("minor must be less than 255"));
```

Comment:
255 seems maybe a bit low for a limit ?

Reply:
OMG! 1am dev seems to have some really bad side effects! 
Fixing that right now :)